js/mocha-tests/package.json

```diff
@@ -0,0 +1,99 @@
+{
+  "name": "mocha-tests",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "lint": "eslint --ignore-path .gitignore .",
```

Comment:
My general personal take on linters is that they should autofix everything they can and only complain about stuff they can't fix themselves. 

In a kata-bootstrap project, I'd rather don't use any styleguide or linting as it hinders people to discover the language.